cmd/kwil-cli/cmds/root.go

```diff
@@ -16,7 +16,39 @@ import (
 	"github.com/kwilteam/kwil-db/cmd/kwil-cli/config"
 )
 
+var longDesc = `Command line interface client for using %s.
+	
+	The %s CLI is a command line interface for interacting with %s.  It can be used to deploy, update, and query databases.
```

Comment:
Probably remove `CLI`? i.e. `The %s CLI` -> `%s`

Reply:
No, the intended statement would be:

Command line interface client for using Kwil.
	
	The Kwil CLI is a command line interface for interacting with Kwil.  It can be used to deploy, update, and query databases.